### DISCUSSION TITLE
[Auth] User must auth to access the docs

### DISCUSSION THREAD

**Discussion by nhattran998** [fuma-nama/fumadocs#1061] [Ideas]
Thank you for the great lib <3 @fuma-nama. 

I want to create a document knowledge base for tech teams in my company but with the company's email. Do you have a plane for that?

**Comment by fuma-nama** [comment]
You can use middleware for auth, it's essentially a Next.js app

**Reply by allandiegoasilva** [reply]
Ok, but a login page is a good idea for implements a good documentation.

**Reply by fuma-nama** [reply]
Use better-auth or anything you want, it's a Next.js app and we don't stop you from doing that
